src/runtime/cmd/kata-monitor/main.go

```diff
@@ -18,7 +18,9 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
-var monitorListenAddr = flag.String("listen-address", ":8090", "The address to listen on for HTTP requests.")
+const defaultListenAddress = "127.0.0.1:8090"
```

Comment:
The idea to leave it running on all the addresses was because the main usage we (or just I? :-P ) had in mind was to have kata-monitor running as a DaemonSet on a Kubernetes cluster, exposing the endpoint to Prometheus.
This anyway could be used on the host directly, so makes sense to be on the safe side and just pass the address if/when running as a DaemonSet.